src/main/java/uk/gov/beis/els/api/categories/televisions/TelevisionsInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.televisions;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotBlank;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.RescaledInternetLabelApiForm;
+import uk.gov.beis.els.categories.televisions.service.TelevisionsService;
+
+@Schema(name = "Televisions and electronic displays internet label")
+public class TelevisionsInternetLabelApiForm extends RescaledInternetLabelApiForm {
+
+  @Schema(description = "Energy efficiency class for SDR content")
```

Comment:
SDR content?

Reply:
that was the wording used in the form, so decided to keep it that way (I'm assuming thats how it appears in legislation)

Reply:
Yeah this is correct, the arrow images uses the Standard Dynamic Range (SDR) rating, rather then the HDR rating. Since both are on the energy label I think it makes sense to specify here